srcpkgs/samba4/template

```diff
@@ -0,0 +1,85 @@
+# Template file for 'samba4'
+pkgname=samba4
+version=4.9.4
+revision=1
+wrksrc="samba-$version"
+build_style=waf
+configure_args="--sysconfdir=/etc/samba --with-configdir=/etc/samba --localstatedir=/var --enable-fhs --with-lockdir=/var/cache/samba --with-piddir=/var/run/samba --with-logfilebase=/var/log/samba --without-pam --without-systemd --with-ads --with-shared-modules=idmap_ad,idmap_rid,idmap_adex,idmap_hash,idmap_tdb2,pdb_tdbsam,pdb_ldap,pdb_ads,pdb_smbpasswd,pdb_wbc_sam,pdb_samba4,auth_unix,auth_wbc,auth_server,auth_netlogond,auth_script,auth_samba4 --enable-cups --without-gettext"
+configure_script=buildtools/bin/waf
+conf_files="/etc/pam.d/samba /etc/samba/smb.conf"
+make_dirs="/etc/samba/private 0750 root root"
+hostmakedepends="pkgconf perl python docbook2x"
+makedepends="readline-devel libcap-devel popt-devel e2fsprogs-devel
+ gnutls-devel python-devel
+ libldap-devel pam-devel acl-devel avahi-libs-devel tdb-devel talloc-devel cups-devel lmdb-devel jansson-devel gpgme-devel libarchive-devel"
+short_desc="SMB/CIFS file, print, and login server for Unix"
+maintainer="Gerardo Di Iorio <arete74@gmail.com>"
+license="GPL-3"
+homepage="http://www.samba.org"
+distfiles="https://download.samba.org/pub/samba/stable/samba-$version.tar.gz"
+checksum=6d98a8d8bcccbe788e4bbb406362e6676311aca711a3f3cc9b3a404bb9ff0b4f
+provides="samba-${version}_${revision}"
+conflicts="samba>=0"
+CFLAGS="-DUSE_SETUIDX=0"
+patch_args="-Np1"
+
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" --bundled-libraries=!asn1_compile,!compile_et"
+	hostmakedepends+=" heimdal-devel"
```

Comment:
please do not include this package

Reply:
heimdal-devel is use only for cross compile. I try to disable heimdal build wthout success.

Reply:
regardless, it conflicts with lots of packages.

Reply:
use system-mit-krb5 with experimental ad-dc